R/print.R

```diff
@@ -4,32 +4,43 @@
 #' Print nicely formatted version of an diffdf object
 #' @param x comparison object created by diffdf().
 #' @param ... Additional arguments (not used)
+#' @param row_limit Max row limit for difference tables (NULL to show all rows)
 #' @param as_string Return printed message as an R character vector?
 #' @examples
 #' x <- subset( iris , -Species )
 #' x[1,2] <- 5
 #' COMPARE <- diffdf( iris, x)
 #' print( COMPARE )
-#' print( COMPARE , "Sepal.Length" )
+#' print( COMPARE, row_limit = 5 )
 #' @export
-print.diffdf <- function(x, ..., as_string = FALSE) {
+print.diffdf <- function(x, row_limit = 10, as_string = FALSE, ...) {
+    if (!is.null(row_limit)) {
+        if (length(row_limit) != 1) {
+            stop("row_limit should have a length of 1")
+        }
+        if (!is.numeric(row_limit)) {
+            stop("row_limit should be a numeric value or NULL")
+        }
+        if (row_limit <= 0) {
+            stop("row_limit should be a positive integer")
+        }
+    }
```

Comment:
So I am being consistent with the rest of the package, where we do not use it. I am very happy to switch to assertthat but wonder if that should be a seperate PR?

Reply:
hold up - row_limit <= 0 is allowed and means all rows - no?

Reply:
that works too - I have calling routines that look for a integer so I used <= 0 but can have that convert to NULL
I didn't use NULL because I thought that that would revert to the default value of 10 but it doesn't.